scripts/README.md

```diff
@@ -1,15 +1,21 @@
-# Scripts: Scripts for variosu Sparkly things
+# Scripts for Various Sparkly Things
 
-Most scripts have usage instructions.
+Most scripts have their own usage instructions within the script themselves.
 
-However some of these scripts require a file called `prodAccountKey.json`.
-
-To obtain this file, go to Firebase and follow these steps:
+These scripts require a Firebase service account key, which you setup once for each environment. To obtain this file, go to the Firebase admin console and follow these steps:
 
 1. Go to Project Settings
 2. Go to Service Accounts
 3. Click `Generate new private key`
-4. Save the key
-5. Move it to `scripts/prodAccountKey.json`
+4. You will be prompted to download a file that looks similar to `PROJECT-ID-firebase-adminsdk-abc12-1234567890.json`
+5. Save it to your `scripts` folder (it is automatically excluded in our default .gitignore file)
+
+WARNING - this file grants passwordless access to ALL data in Firebase, including delete access. Treat the file like a private key and keep it safe.
```

Comment:
```suggestion
**WARNING:** this file grants passwordless access to ALL data in Firebase, including delete access. Treat the file like a private key (because it is one!) and keep it safe.
```

Reply:
fix 6c6c049